### PR TITLE
Add build dependencies for Mockito testcase

### PR DESCRIPTION
**Pull request by sophiaxu0424**
- As required from issue 5073: https://github.com/adoptium/aqa-tests/issues/5073 for adding new Mockito test case
- Add 3 more dependencies accordingly.

### CODE REVIEW

**Review by LongyuZhang — APPROVED** [review]
LGTM

**Review by karianna — APPROVED** [review]

File: scripts/getDependencies.pl [hunk 1]
```diff
@@ -94,6 +94,21 @@
...
+		url => 'https://repo1.maven.org/maven2/org/mockito/mockito-core/5.9.0/mockito-core-5.9.0.jar',
+		fname => 'mockito-core.jar',
+		sha1 => 'faa88b96db3aeb96a93e83dec7491345bfbfc414'
+	},
+	byte_buddy => {
+		url => 'https://repo1.maven.org/maven2/net/bytebuddy/byte-buddy/1.14.12/byte-buddy-1.14.12.jar',
+		fname => 'byte-buddy.jar',
+		sha1 => '6e37f743dc15a8d7a4feb3eb0025cbc612d5b9e1'
+	},
+	maven_jar_plugin => {
```
Comment (karianna) [inline scripts/getDependencies.pl:107]: @sophiaxu0424 My bad for missing this, I don't think we need this JAR file, can you submit another PR to remove this?
